release/preview/centos7/docker/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM centos:${fromTag}
 
 ARG VCS_REF="none"
 ARG PS_VERSION=6.1.0~rc.1
-ARG IMAGE_NAME=microsoft/powershell:centos7
+ARG IMAGE_NAME=mcr.microsoft.com/powershell:centos7
```

Comment:
Is this change of labels purely cosmetic? or some functionality/UX depends on it?

Reply:
This is more like something that a customer is supposed to be able to query.  Right now it's mainly cosmetic.

Here is a site that uses the information:  https://microbadger.com/images/microsoft/powershell